build-tools/packages/build-tools/src/type-test-generator/main.ts

```diff
@@ -54,6 +54,7 @@ if (!existsSync(`${previousBasePath}/package.json`)) {
 		`${previousBasePath} not found. You may need to install the package via pnpm install.`,
 	);
 }
+const alphaFilePath = `${previousBasePath}/dist/${previousPackageName}-alpha.d.ts`;
```

Comment:
A potential issue with making the logic dynamic based on the presence of the trimmed rollup is that we are already generating these rollups, but not yet using them in the packages we publish. This means that we could begin using the trimmed alpha types for comparison, while we are still making our internal types publicly available. 

I would recommend updating the typetest config (package.json's `typeValidation` properties) to accept an optional parameter that specifies a type-rollup file to be used. When specified, we can look for that file and throw an error if it doesn't exist. If not specified, we use `index.d.ts` (+ all of the other d.ts files under dist, since `index.d.ts` doesn't contain all of the necessary info).